geode-redis/src/distributedTest/resources/org/apache/geode/redis/springRedisTestApplication/application.properties

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2019-Present Pivotal Software, Inc. All rights reserved.
+#
+# This program and the accompanying materials are made available under the terms of the under the Apache License, Version
+# 2.0 (the "License”); you may not use this file except in compliance with the License. You may obtain a copy of the
+# License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
```

Comment:
I don't know very much about copyrights... is there any weirdness around having this app inside our module or having the Pivotal copyright there, etc.?